### PR TITLE
Add fragments and navigate

### PR DESCRIPTION
**Pull request by corinaorasan**
## Description
Create a navigation graph for the trivia app, add the Title Screen and the Game Screens, connect them with an action, and navigate to the game screen when the play button is pressed.

## Steps to Test or Reproduce
1.Start
2.Press play button
## Risks
No risks

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -63,4 +64,5 @@ dependencies {
     // Navigation
     implementation "android.arch.navigation:navigation-fragment-ktx:$version_navigation"
     implementation "android.arch.navigation:navigation-ui-ktx:$version_navigation"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (deniscoman) [inline app/build.gradle]: Extract version number in a variable like we have on the other libraries

File: app/src/main/java/com/example/android/navigation/GameFragment.kt [hunk 2]
```diff
@@ -60,52 +61,51 @@ class GameFragment : Fragment() {
...
             val checkedId = binding.questionRadioGroup.checkedRadioButtonId
-            // Do nothing if nothing is checked (id == -1)
             if (-1 != checkedId) {
-                var answerIndex = 0
-                when (checkedId) {
-                    R.id.secondAnswerRadioButton -> answerIndex = 1
-                    R.id.thirdAnswerRadioButton -> answerIndex = 2
-                    R.id.fourthAnswerRadioButton -> answerIndex = 3
-                }
+                changeAnswerIndex(checkedId)
```
Comment (deniscoman) [inline app/src/main/java/com/example/android/navigation/GameFragment.kt]: We should have a better name for this function.

**Review by deniscoman — CHANGES_REQUESTED** [review]

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,7 +1,6 @@
-#Thu Feb 11 17:08:34 EET 2021
+#Mon Feb 15 10:58:12 EET 2021
```
Comment (AlexandraDamaschin) [inline gradle/wrapper/gradle-wrapper.properties]: We should not track this changes on Git as they are irrelevant for code reviews

**Comment by AlexandraDamaschin** [comment]
Learn more about squash and apply them to this PR as you have loads of duplicated commits with the same message: 
image

**Review by AlexandraDamaschin — CHANGES_REQUESTED** [review]
Apply git changes to make your pr nicer.

**Review by AlexandraDamaschin — APPROVED** [review]

**Pull request merged**